docs/user-guide/mvd-using.md

```diff
@@ -74,35 +93,29 @@ Use the Languages setting in the Preferences panel to change the desktop languag
 
 ## Zowe Desktop application plug-ins
 
-Application plug-ins are applications that you can use to access the mainframe and to perform various tasks. Developers can create application plug-ins using a sample application as a guide. The following application plug-ins are installed by default:
+Application plugins are applications that you can use to access the mainframe and to perform various tasks. Developers can create application plugins using a sample application as a guide. The following application plug-ins are installed by default:
 
 ### Hello World Sample
```

Comment:
plugin --> plugins

You should mention that we currently have Sample plugins for Iframes, React, and Angular (each with their own Git link)
sample-iframe-app, sample-angular-app, sample-react-app etc